doc/Language/create-cli.pod6

```diff
@@ -12,7 +12,7 @@ The default command line interface of Raku scripts consists of three parts:
 =head2 Parsing the command line parameters into a L<capture|/type/Capture>
 
 This looks at the values in L<@*ARGS|/language/variables#index-entry-@*ARGS>,
-interprets these according to some policy, and creates a L<Capture/type/Capture>
+interprets these according to some policy, and creates a L<capture|/type/Capture>
```

Comment:
Can you please keep the capital for Capture here?

Reply:
Done :)